codeHF/PlotEfficiency_recoStep.C

```diff
@@ -0,0 +1,383 @@
+// Computation and plotting of reconstruction efficiency step-by-step
+// Four steps defined: kRecoHFFlag, kRecoTopol, kRecoCand, kRecoPID
+// Macros input: AnalysisResults.root from O2 with TH2F (pt vs y) distributions for generated and reconstructed candidates
+// .L PlotEfficiency_recoStep.C
+// PlotEfficiencySbyS("InputName.root","particlename",true);
```

Comment:
It should be mentioned that `particles` can contain multiple particle strings.
What is `effSbyS`?

Reply:
Done.
effSbyS means "efficiency Step-by-Step" to distinguish it from the efficiency done at a defined step

Reply:
I don't see the use of `effSbyS`. The macro does not seem to do anything in case it is `false`.

Reply:
I introduced it when I was developing the code in the same macro of `PlotEfficiency.C`. Since we have a separate macro for the efficiency step-by-step I remove this boolean.